packages/amplify-cli-core/resources/overrides-resource/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-amplify/cli-extensibility-helper": "2.3.33-signinUpdate.1"
+    "@aws-amplify/cli-extensibility-helper": "^2.0.0"
```

Comment:
should this be cdk version ?

Reply:
This is a template file for overrides. I think we can keep this as it is because anyways we have to manually change package.json to the latest tagged release.

Reply:
We should move templates for overrides to `^3.0.0-cdkv2.0` and keep it there until beta/GA. 
`^3.0.0-cdkv2.0` would automatically absorb `3.0.0-cdkv2.0`, `3.0.0-cdkv2.1` and `3.0.0`.
(this should be separate PR from merge if we haven't done it yet).